.env.example

```
@@ -16,6 +16,11 @@ SANDBOX_SUPERVISOR_URL=http://127.0.0.1:7091
 # Dedicated credentials (distinct from BETTER_AUTH_SECRET). Required for Docker sandboxes /
 # screen proxy outside local development.
 SANDBOX_SUPERVISOR_TOKEN=replace-with-32-plus-character-supervisor-token
+# Per-bot-computer ceilings. Defaults 2g / 2 CPUs / 2048 pids; "0", "none" or "unlimited" removes
+# one. Memory also caps swap, and must be at least 6m (Docker's own minimum).
+RAKAZO_COMPUTER_MEMORY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RAKAZO_COMPUTER_MEMORY key should go before the SANDBOX_SUPERVISOR_TOKEN key

(UnorderedKey)